config/config.exs

```diff
@@ -12,9 +12,9 @@ config :git_hooks,
         {:cmd, "mix credo"},
         {:cmd, "mix sobelow"},
         {:cmd, "mix knigge.verify"},
-        {:cmd, "mix test --trace"},
-        {:cmd, "mix dialyzer"},
-        {:cmd, "mix check.updates"}
+        # {:cmd, "mix test --trace"},
```

Comment:
I think this should be uncommented

Reply:
yes, but there is an issue with this I created a separate ticket for that however it needs to be uncommented to not affect hooks usage in other areas than governance for now

Reply:
done